stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -308,7 +317,18 @@ public class MessageListView : ConstraintLayout {
         return adapter.itemCount - 1
     }
 
+    public fun setLoadingMore(loadingMore: Boolean) {
+        if (::loadMoreListener.isInitialized) {
```

Comment:
Because the `loadMoreListener` doesn't depend on the UI Components, we could initialize it before and don't have `lateinit var`
`EndlessScrollListener` should provide some way to update the `loadMoreThrshold`.

Reply:
Moved loadMore initialization to the class field